src/javascript/app_2/App/Components/Elements/ContractDrawer/contract-drawer.jsx

```diff
@@ -0,0 +1,74 @@
+import classNames            from 'classnames';
+import PropTypes             from 'prop-types';
+import React, { Component }  from 'react';
+import { withRouter }        from 'react-router';
+import Localize              from 'App/Components/Elements/localize.jsx';
+import { Icon, IconBack }    from 'Assets/Common';
+import routes                from 'Constants/routes';
+import ProfitLossCardContent from 'Modules/Reports/Components/profit-loss-card-content.jsx';
+import MarketSymbolIconRow   from 'Modules/Reports/Components/market-symbol-icon-row.jsx';
+import ContractCardBody      from './contract-card-body.jsx';
+import ContractCardFooter    from './contract-card-footer.jsx';
+import ContractCardHeader    from './contract-card-header.jsx';
+import ContractCard          from './contract-card.jsx';
+import ContractAudit         from '../ContractAudit/contract-audit.jsx';
+import Money                 from '../money.jsx';
+
+class ContractDrawer extends Component {
+    getBodyContent () {
+        const { buy_price } = this.props.contract_info;
```

Comment:
Why not include things like `profit`, `payout` or `currency` too and use them instead?